pages/automatedtestingpython.md

```diff
@@ -72,10 +72,19 @@ Finally:
 
 ## MacOS/Ubuntu - Installing Testing Development Environment
 ### Installing Nation
-WIP - Needs some love.
+The process is similar to installing Nation for Windows.
 
 ### Installing Python, Pip, Selenium, Geckodriver, and Firefox
-WIP - Needs some love.
+1. Install Python 3 with `brew install python3` or download Python from https://www.python.org/downloads/release/python-352/)
```

Comment:
Hi @mappuji,  `brew` is only specific to mac. What I add on is only work for Mac because I familiar with Mac but not Ubuntu. Other interns who familiar with using Ubuntu may add some details later if needed. 
Another thing, if you really want, I can take time to work on ubuntu and update the document. 
P/S: I have Linux on my computer.